examples/Vendored XCFramework Example/BananaLib/DynamicFramework/CoconutLib.xcframework/tvos-arm64/CoconutLib.framework/BCSymbolMaps/15C7CC08-7EDC-3995-9AFD-42F51335C1E2.bcsymbolmap

```diff
@@ -0,0 +1,11 @@
+BCSymbolMap Version: 2.0
```

Comment:
nice very helpful to have one of those.

Reply:
also TIL these include full paths to local dev environment 🤦 non-deterministic builds ftw...